zerocode-maven-archetype/README.txt

```diff
@@ -0,0 +1,22 @@
+To install archetype locally:
+    Navigate to zerocode-maven-archetype on your machine. Run "mvn install" from this directory
+    
+To generate anew archetype-based project:
+    Navigate to the directory that will house the project. Run 
+    "mvn archetype:generate 
+    -DarchetypeGroupId=com.myproject 
+    -DarchetypeArtifactId=zerocodeArchetype 
+    -DarchetypeVersion=1.0-SNAPSHOT 
+    -DgroupId=com.myproject 
+    -DartifactId=zerocodeArchetypeTest"
+    
+    The generic command format is:
+    "mvn archetype:generate -DarchetypeGroupId=<custom-archetype group id e.g.>
```

Comment:
The archetypeGroupId appears to be set to `com.myproject`, I'd personally change this to reflect something Zerocode related as this is the group ID of the archetype as opposed to the group ID of the newly created project.